novawallet/Modules/Staking/StakingType/StakingTypePresenter.swift

```diff
@@ -19,6 +19,7 @@ final class StakingTypePresenter {
     private var method: StakingSelectionMethod?
     private var selection: StakingTypeSelection
     private var hasChanges: Bool = false
+    private var recommendedValidators: PreparedValidators?
```

Comment:
Not sure if we are really need this. As I see we can just pass information whether the selected validation are actually recommended from the CustomValidators module. What do you think?